moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h

```diff
@@ -200,7 +200,7 @@ moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_n
  *
  * @return was the construction successful?
  */
-bool constructConstraints(XmlRpc::XmlRpcValue& params, moveit_msgs::msg::Constraints& constraints);
+// bool constructConstraints(XmlRpc::XmlRpcValue& params, moveit_msgs::msg::Constraints& constraints);
```

Comment:
Add TODO with missing dependency. Was this removed because xmlrpc is not available? If so, it looks like it is used in the [ros1_bridge](https://github.com/ros2/ros1_bridge/blob/7139469ae3ad7f6cff67baf09086b47f554f47e8/src/dynamic_bridge.cpp#L495)

Reply:
We can actually use the rclcpp parameters to fix these kind of things, or at least, the way that I'm following.

Reply:
@v4hn spent a lot of time on this feature. Perhaps you and he can collaborate on the best path for porting this. In the mean time, any time you comment out a feature, please add comments explaining why the feature is being removed and what steps would be necessary before re-enabling the feature